domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterSingletonImpl.kt

```diff
@@ -26,6 +26,8 @@ class PlatformParameterSingletonImpl @Inject constructor() : PlatformParameterSi
     return object : PlatformParameterValue<String> {
       override val value: String
         get() = parameter.string
+      override val isSynced: Boolean
+        get() = parameter.isSynced
```

Comment:
```suggestion
      override val value = parameter.string
      override val isSynced = parameter.isSynced
```

Can be simplified. Ditto for the other updates in this class.

Reply:
This has been updated as part of the "removing getters" changes.